IdentityCore/src/cache/MSIDMacTokenCache.m

```diff
@@ -92,7 +92,13 @@ - (nullable NSData *)serialize
         {
             NSMutableData *data = [NSMutableData data];
 
-            NSKeyedArchiver *archiver = [[NSKeyedArchiver alloc] initForWritingWithMutableData:data];
+            NSKeyedArchiver *archiver;
+#if TARGET_OS_UIKITFORMAC
+            archiver = [[NSKeyedArchiver alloc] initRequiringSecureCoding:YES];
```

Comment:
it seems this API is supported since macOS 10.13. Should we switch to it on all macOS 10.13+ platforms?

Reply:
done